### PR TITLE
one way to fix akka logging

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
One way to fix https://github.com/typesafehub/ssl-config/issues/6 though it certainly is not "nice".
This is caused by the desire to keep ssl-config free from dependencies on akka.

**Comment by hepin1989** [comment]
like the Play way:)

**Comment by ktoso** [comment]
Build passed https://travis-ci.org/typesafehub/ssl-config/builds/98075738
Please have a look @rkuhn, good enough solution for logging here?
I don't really see a better way given that ssl-config-core should not depend on Akka.

File: ssl-config-akka/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala.orig [hunk 1]
```diff
@@ -0,0 +1,184 @@
+/*
```
Comment (rkuhn) [inline ssl-config-akka/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala.orig]: this file does not belong …

Comment (ktoso) [inline ssl-config-akka/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala.orig]: thanks!

**Comment by ktoso** [comment]
Now solved via explicit logger factory passing. Verbose but works in any case and style of logger.

**Pull request merged**